shared/common-adapters/user-proofs.shared.js

```diff
@@ -83,5 +83,6 @@ export function proofNameStyle (proof: Proof) {
   return {
     color: proofColor(proof),
     ...(proof.meta === metaDeleted ? globalStyles.textDecoration('line-through') : {}),
+    ...(['btc', 'pgp'].includes(proof.type) ? {fontSize: 13} : {}),
```

Comment:
smaller rendition for keys else they wrap and look kinda crap. also matches what we do on the website (idea from coyne)